scripts/restream.py

```diff
@@ -1,33 +1,28 @@
-# invoke from root as python -m scripts.restream
 import sys
+import os
 from subprocess import Popen, PIPE
 import time
 import re
 
-with open("config.stream.template.json", "r") as content_file:
-    config_base_file = content_file.read()
+from nestris_ocr.config_class import Config
 
 
-RESOLUTIONS = {
-    "1080": [1920, 1080],
-    "720": [1280, 720],
-    "480": [852, 480],
-    "360": [640, 360],
-}
+TEMPLATE_FILE = "scripts/config.stream.template.json"
 
 CAP_RATIOS = [1233 / 1920, 1]  # based on the stencil dimensions for 1080p
 
+# entries as [VLC_BRODCAST_PORT, OCR_SEND_PORT]
 PLAYER_SETTINGS = [
     [8081, 4001],
     [8082, 4002],
 ]
 
 
-def setupPlayer(twitch_name, player_num):
-    local_stream_port, ocr_dest_port = PLAYER_SETTINGS[player_num - 1]
+def setupPlayer(twitchName, playerNum):
```

Comment:
Why are they camelCase now 😂 My complaint was that things should be in camel_case, so `def setup_player`. In modern Python, we should have either PascalCase (for class names etc.), ALL_CAPS (constants), or camel_case (variable names, function names). 

I should have been more specific to prevent this from happening 😭

Reply:
LOL! 😅 Fine I'll update all of the names again tomorrow

Reply:
Yeah its 99% my fault for having camelCase in the original code base. Blake has graciously helped rename a lot of stuff, but we want to move towards what he said. The code is a mish-mash atm and we want new code to comply.

Reply:
All names following in script following correct convention now 👍 🎉